src/sidebar/services/annotations.js

```diff
@@ -117,7 +117,12 @@ export default function annotationsService(api, store) {
     if (metadata.isPageNote(annotation)) {
       store.selectTab('note');
     } else if (metadata.isAnnotation(annotation)) {
-      store.selectTab('annotation');
+      // if this is a doodle, select the doodle tab
+      if (annotation.$doodle) {
+        store.selectTab('orphan');
```

Comment:
It looks like you added a doodle tab in ui-constants.js, did you want to select that tab here?

Reply:
I started to add the tab but I think thats best done as a separate PR also. It's a little more complicated than I thought it would be. This code was just a holdover from that.

Reply:
Makes sense, do you have a JIRA issue for the new tab?